styles/styles.css

```
@@ -0,0 +1,316 @@
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 700;
+    src: url("fonts/ptsans-700.woff2") format("woff2");
+    font-display: swap;
+}
+
+body{
+    font-family: "PT Sans",sans-serif;
+    font-size: 18px;
+    line-height: 21px;    
+    background-color: #f2f2f2;
+}
+
+.visually-hidden{
+    visibility: hidden;
+}
+
+.page-header{
+    background-color: #ffffff;
+    color:#000000;
+}
+
+.navigation-link{
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 26px;
+    text-decoration: none;
+}
+
+.counter{
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 13px;
+    color: #000000;/*цвет должен быть #ffffff*/
```

Если должен значит делай, и сразу ставь фон :)
```suggestion
  color: #ffffff;
  background-color: #7db54f;
```